Monika After Story/game/script-topics.rpy

```diff
@@ -12905,3 +12905,78 @@ label monika_enjoyingspring:
+    m 1esd "But they also tend to be somewhat selfish and uncooperative, always wanting to go their own way, without much considerations."
+    if persistent._mas_pm_cares_about_dokis:
+        m 1lksdrb "Without wanting to disrespect her, it's sort of what Nastuki is, don't you agree?"
+    else:
+        m 1tsb "It clearly is Natsuki."
+    m 2esd "I mean, do you remember when she threatened to quit the club just because it was going in a direction that she didn't like?"
+    m 2rfc "Although she had her reasons, it was impulsive and very inconsiderate of her."
+    if persistent.blood_type == "B":
+        m 1wuo "Ah, and bear in mind that those are just superstitions without proof whatsoever of their validity."
+        m 1hksdrb "I know this is your blood type we are talking about, but don't mistake it for a personality test."
+        m 1eka "You're way more awesome and complex than that [player]."
+    m 2dsd "But I digress."
+    m 1eub "Next on our list, we have the O group!"
+    m 4eub "Those ones are generally very agreeable and easygoing, they are very intuitive and leading, especially when it concern other peoples."
+    m 1eud "But they are also quite unpredictable and unpunctual, sometimes even jealous or ruthless."
+    m 1lud "It does not fit perfectly, but some of the traits are shared by Sayori."
+    m 1eka "She really liked take care of other people, especially you."
+    m 4eud "Back in the days, before you even came to the club, when a fight would broke out, she'd instantly know what was the right words to calm everyone."
+    m 2dsc "But she also had many issues because of her unpunctuality."
+    if persistent._mas_pm_cares_about_dokis:
+        m 2ekd "When you know what caused it, you can't help but feeling sorry for her."
+        m 2dkc "She really cared about things, but her depression just wouldn't leave her alone."
+    else:
+        m 2tkc "I'm almost feeling sorry for the MC, having to put up with someone that unreliable."
+        m 3efd "Even if she had her reasons, she never tried to change it, to make things better. And a lot of people ended up suffering unnecessarily because of that"
+        m 1dsc "Sorry, there's no need to be angry about that, it's behind us now."
+    if persistent.blood_type == "O":
+        m 1wuo "Ah, and bear in mind that those are just superstitions without proof whatsoever of their validity."
+        m 1hksdrb "I know this is your blood type we are talking about, but don't mistake it for a personality test."
+        m 1eka "You're way more awesome and complex than that [player]."
+    m 3eub "Anyway, that leaves us with only one left."
+    m 1tsu "I saved the best for last, ehehe~."
+    m 1dua "The A group correspond to..."
+    m 1hub "Me, of course!"
+    m 4wub "Peoples that have the A type are responsible, perfectionists, wise, earnest, neat and much more."
```

Comment:
What I was trying to do is to have Monika self-congratulate herself only to take the backlash of recalling all of the default also associated with this group. So yes, it does not fit perfectly but it's because Monika is being very inclusive just to feel good about herself for a short time. I don't mean it in an arrogant way, but more in a joke-like way.